src/clang_parser.cpp

```diff
@@ -298,6 +299,11 @@ void ClangParser::parse(ast::Program *program, BPFtrace &bpftrace, std::vector<s
               struct_name = ptypestr;
             remove_struct_prefix(struct_name);
 
+            if (clang_Type_getSizeOf(type) < 0) {
+              std::cerr << "Can't get size of '" << ptypestr << "::" << ident << "', please provide proper definiton." << std::endl;
```

Comment:
This error message might not be actionable in most cases (structs defined in kernel headers for example). I think it's fine keeping it as is for now, but we might want to improve it in the future.